### PR TITLE
Able to clean storage items with soap

### LINKED ISSUE #64258: Couldn't clean toolbox with soap
Reporting client version: 514.1571

## Round ID:
[176902](https://scrubby.melonmesa.com/round/176902)

## Testmerges:
- [Adds Thieves, a very low level antagonist.](https://www.github.com/tgstation/tgstation/pull/64144)

## Reproduction:

1. Left and right-click on a toolbox with combat mode both on and off
2. Soap either goes into the toolbox or the toolbox's storage is opened

### PR DESCRIPTION
**Pull request by etherware-novice**
## About The Pull Request
You can clean storage items if you have combat mode on, while still being able to insert with combat off.

## Why It's Good For The Game
closes #64258 
It is a strange thing to leave out, especially considering toolbox combat leaving a ton of bloody toolboxes everywhere.

## Changelog

:cl:
fix: you can now clean storage items (toolbox) with soap
/:cl:

### CODE REVIEW

File: code/game/objects/items/clown_items.dm [hunk 1]
```diff
@@ -150,6 +150,9 @@
 		return
 	return ..()
 
+/obj/item/soap/attackby_storage_insert(datum/storage, atom/storage_holder, mob/living/user)
+	. = ..()
+	return !(user?.combat_mode)  // only cleans a storage item if on combat
```
Comment (Mothblocks) [inline code/game/objects/items/clown_items.dm:155]:
```suggestion
	return !user?.combat_mode  // only cleans a storage item if on combat
```

**Comment by Mothblocks** [comment]
Not great but I'm not sure right-click would be any more intuitive here

**Comment by etherware-novice** [comment]
you still need to confirm the review i think

**Comment by Mothblocks** [comment]
auto merge merges when ci passes, it didnt because i had just made an edit

**Pull request merged**